src/demo/DemoTestActivitiesService.java

```diff
@@ -117,6 +119,54 @@ public static void testGetActivitiesForEntity() throws MambuApiException {
 		List<JSONActivity> jsonActivities = activitiesService.getActivities(fromDate, toDate, mambuEntity, entityId);
 
 		// Print results
+		printMambuEntityAndActivitiesDetails(mambuEntity, entityId, jsonActivities, fromDate, toDate);
+	}
+	
+	/**
+	 * Tests getting activities paginated for a client 
+	 * 
+	 * @throws MambuApiException 
+	 * 
+	 */
+	public static void testGetActivitiesWithPaginationForClient() throws MambuApiException {
+
+		String methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println(methodName = "\nIn " + methodName);
+
+		ActivitiesService activitiesService = MambuAPIFactory.getActivitiesService();
+
+		Class<Client> mambuEntity = Client.class;
+
+		Calendar toDate = Calendar.getInstance(); 
+
+		Calendar fromDate = Calendar.getInstance();
+		fromDate.add(Calendar.DAY_OF_MONTH, -10);
+		
+		String entityId = DemoUtil.getDemoClient().getEncodedKey();
+		if (entityId == null) {
+			System.out.println("WARNING: No Activities for Client entity to retrieve");
```

Comment:
Can we have a more meaningful message here? For instance, I think it would be better to specify that there are no activities for this client because its encoded key is null or because the client does not exist at all. What do you think?